ctapipe/io/simteleventsource.py

```diff
@@ -274,6 +288,113 @@ def apply_simtel_r1_calibration(
     return r1_waveforms, selected_gain_channel
 
 
+@unique
+class AtmosphereProfileKind(Enum):
+    """
+    choice for which kind of atmosphere density profile to load if more than
+    one is present"""
+
+    #: Don't load a profile
+    NONE = auto()
+
+    #: Use a TableAtmosphereDensityProfile
+    TABLE = auto()
+
+    #: Use a FiveLayerAtmosphereDensityProfile
+    FIVELAYER = auto()
+
+    #: Try TABLE first, and if it doesn't exist, use FIVELAYER
+    AUTO = auto()
+
+
+def read_atmosphere_profile_from_simtel(
+    simtelfile: Union[str, Path, SimTelFile], kind=AtmosphereProfileKind.AUTO
+) -> Optional[TableAtmosphereDensityProfile]:
+    """Read an atmosphere profile from a SimTelArray file as an astropy Table
+
+    Parameters
+    ----------
+    simtelfile: str | SimTelFile
+        filename of a SimTelArray file containing an atmosphere profile
+    kind: AtmosphereProfileKind | str
+        which type of model to load. In AUTO mode, table is tried first,
+        and if it doesn't exist, fivelayer is used.
+
+    Returns
+    -------
+    Optional[TableAtmosphereDensityProfile]:
+        Profile read from a table, with interpolation
+
+    """
+    if not isinstance(kind, AtmosphereProfileKind):
+        raise TypeError(
+            "read_atmosphere_profile_from_simtel: kind should be a AtmosphereProfileKind"
+        )
+
+    profiles = []
+
+    if kind == AtmosphereProfileKind.NONE:
+        return None
+
+    if isinstance(simtelfile, (str, Path)):
+        context_manager = SimTelFile(simtelfile)
+        Provenance().add_input_file(
+            filename=simtelfile, role="ctapipe.atmosphere.AtmosphereDensityProfile"
+        )
+
+    else:
+        context_manager = nullcontext(simtelfile)
+
+    with context_manager as simtel:
+
+        if (
+            not hasattr(simtel, "atmospheric_profiles")
+            or len(simtel.atmospheric_profiles) == 0
+        ):
+            return []
+
+        for atmo in simtel.atmospheric_profiles:
+
+            metadata = dict(
+                obs_level=atmo["obslevel"] * u.cm,
+                atmo_id=atmo["id"],
+                atmo_name=atmo["name"],
+                htoa=atmo["htoa"],  # what is this?,
```

Comment:
We should answer that question

Reply:
Ah, I found it I think:

```cpp
corsika/atmo.c:
2588:   double htoa = 113e5;  /* Height at top of atmosphere */
```
at least in the version I have downloaded.  I'll update it in the file